AIX build breaks

After #4720, build is broken on AIX.

/opt/IBM/openxlC/17.1.2/bin/ibm-clang_r -c -O2 -DSMALL_MATRIX_OPT -DMAX_STACK_ALLOC=2048 -fopenmp -DHAVE_P10_SUPPORT -Wall -DF_INTERFACE_IBM -fPIC -DDYNAMIC_ARCH -DSMP_SERVER -DUSE_OPENMP -DNO_WARMUP -DMAX_CPU_NUMBER=16 -DMAX_PARALLEL_NUMBER=1 -DBUILD_BFLOAT16 -DBUILD_SINGLE=1 -DBUILD_DOUBLE=1 -DBUILD_COMPLEX=1 -DBUILD_COMPLEX16=1 -DVERSION=\"0.3.27.dev\" -DUSE_OPENMP -fopenmp -fno-integrated-as -m64 -UASMNAME -UASMFNAME -UNAME -UCNAME -UCHAR_NAME -UCHAR_CNAME -DASMNAME=cblas_sbgemm_batch -DASMFNAME=cblas_sbgemm_batch_ -DNAME=cblas_sbgemm_batch_ -DCNAME=cblas_sbgemm_batch -DCHAR_NAME=\"cblas_sbgemm_batch_\" -DCHAR_CNAME=\"cblas_sbgemm_batch\" -DNO_AFFINITY -I.. -I. -DBFLOAT16 -UDOUBLE  -UCOMPLEX -DCBLAS gemm_batch.c -o cblas_sbgemm_batch.o

**gemm_batch.c:117:6: error: conflicting types for 'cblas_sbgemm_batch'**
  117 | void CNAME(enum CBLAS_ORDER order, enum CBLAS_TRANSPOSE *  transa_array, enum CBLAS_TRANSPOSE * transb_array,
        |      ^
<command line>:27:15: note: expanded from macro 'CNAME'
   27 | #define CNAME cblas_sbgemm_batch
        |               ^
../cblas.h:446:6: note: previous declaration is here
  446 | void cblas_sbgemm_batch(OPENBLAS_CONST enum CBLAS_ORDER Order, OPENBLAS_CONST enum CBLAS_TRANSPOSE * TransA_array, OPENBLAS_CONST enum CBLAS_TRANSPOSE * TransB_array, OPENBLAS_CONST blasint * M_array, OPENBLAS_CONST blasint * N_array, OPENBLAS_CONST blasint * K_array,
         |      ^
1 error generated.
make[1]: *** [Makefile:2423: cblas_sbgemm_batch.o] Error 1

--- a/interface/gemm_batch.c
+++ b/interface/gemm_batch.c
@@ -118,8 +118,8 @@ void CNAME(enum CBLAS_ORDER order, enum CBLAS_TRANSPOSE *  transa_array, enum CB
 	   blasint * m_array, blasint * n_array, blasint * k_array,
 #ifndef COMPLEX
 	   FLOAT * alpha_array,
-	   FLOAT ** a_array, blasint * lda_array,
-	   FLOAT ** b_array, blasint * ldb_array,
+	   IFLOAT ** a_array, blasint * lda_array,
+	   IFLOAT ** b_array, blasint * ldb_array,
 	   FLOAT * beta_array,
 	   FLOAT ** c_array, blasint * ldc_array, blasint group_count, blasint * group_size) {
 #else

